media-fonts/smiley-sans/smiley-sans-1.1.1.ebuild

```
@@ -7,7 +7,7 @@ inherit font
 
 MY_P="${PN}-v${PV}"
 
-DESCRIPTION="得意黑 Smiley Sans: an open-source italic sans-serif Chinese font"
+DESCRIPTION="Smiley Sans: an open-source italic sans-serif Chinese font"
```

Looks like "得意黑" = "Deyi Hei" is the name of the font family, see for example: https://eng.fontke.com/search/font/%E5%BE%97%E6%84%8F%E9%BB%91%EF%BC%88Deyi%20Hei%EF%BC%89/
```suggestion
DESCRIPTION="Deyi Hei Smiley Sans: an open-source italic sans-serif Chinese font"
```